Modules/BenchPress.Azure/Public/Confirm-ApiManagementApi.ps1

```diff
@@ -0,0 +1,56 @@
+# INLINE_SKIP
+using module ./../Classes/ConfirmResult.psm1
+
+. $PSScriptRoot/../Private/Connect-Account.ps1
+# end INLINE_SKIP
+
+function Confirm-ApiManagementApi {
+  <#
+    .SYNOPSIS
+      Confirms that an API Management API exists.
+
+    .DESCRIPTION
+      The Confirm-AzBPApiManagementApi cmdlet gets an API Management API using the specified API, API Management
+      Service, and Resource Group names.
+
+    .PARAMETER ResourceGroupName
+      Specifies the name of the resource group under which an API Management service is deployed.
```

Comment:
For these parameter descriptions, you mind using the same format we've had for the other cmdlets?

Ex: We usually don't say `Specifies...`, we just call it `The name of <x>`

Reply:
I had mentioned this in another comment (for a different reason), but these are copy/paste from the Azure PowerShell documentation for the cmdlets that we're calling. My preference would be to go the other way with it and update the other existing files to match so that what we are describing is consistent with the Azure PowerShell documentation.